samples/bluetooth/central_bas/prj.conf

```diff
@@ -16,4 +16,12 @@ CONFIG_BT_SCAN=y
 CONFIG_BT_SCAN_FILTER_ENABLE=y
 CONFIG_BT_SCAN_UUID_CNT=1
 
+CONFIG_BT_SETTINGS=y
+CONFIG_FLASH=y
+CONFIG_FLASH_PAGE_LAYOUT=y
+CONFIG_FLASH_MAP=y
+CONFIG_FCB=y
```

Comment:
Should we use NVS by default? @nvlsianpu

Reply:
@nvlsianpu @carlescufi Is there an easier way to add flash storage support? Seems like there is the same 7 configs added everytime, maybe we could get some sensible defaults in place?

Reply:
> 
> 
> Should we use NVS by default? @nvlsianpu

I discussed it offline with @nvlsianpu and we decided that, as a first step, it would be best to enable NVS in the upstream (in Bluetooth samples that support bonding). If all is well and the upstream PR is merged, I will make a similar one in this repo.

I would prefer to remove NVS from the scope of this PR.

Reply:
> I would prefer to remove NVS from the scope of this PR.

Agreed, nothing needs to be done for NVS in this PR.